traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/ds/DeliveryService.java

```diff
@@ -590,6 +610,14 @@ public String getRoutingName() {
 		return routingName;
 	}
 
+	public String getTopology() {
+		return topology;
+	}
+
+	public boolean hasRequiredCapabilities(final Set<String> capabilities) {
+		return this.requiredCapabilities.containsAll(capabilities);
```

Comment:
I think this logic might be backwards. The _server_ needs to have _all_ the capabilities that the DS requires. So I think this should be `capabilities.containsAll(requiredCapabilities)`, and consider renaming `capabilities` to `serverCapabilities` to make it more clear.

Reply:
Logic flipped in 01f871c17